redaxo/src/core/lib/console/application.php

```diff
@@ -54,7 +54,20 @@ private function loadPackages(rex_console_command $command)
         // Some packages requires a working db connection in their boot.php
         // in this case if no connection is available, no commands can be used
         // but this command should be always usable
-        if ('db:set-connection' === $command->getName()) {
+        if ($command instanceof rex_command_standalone) {
+            return;
+        }
+
+        // Loads only setup packages
+        // This is useful for any kind of pre-setup commands
+        // there a packages which are needed during the setup e.g. backup
+        if (rex::isSetup() && $command instanceof rex_command_only_setup_packages) {
```

Comment:
das isSetup muss glaube ich nur um das foreach -> enlist.
Das booten muss hingegen immer passieren.
in der console.php wird ja nur enlist aufgerufen (wenn nicht setup)